modules/library/opengis/src/main/java/org/opengis/feature/simple/SimpleFeature.java

```diff
@@ -143,7 +143,7 @@
      *
      * @param values The attribute values to set.
      */
-    void setAttributes(Object[] values);
```

Comment:
Dangerous conversion here, will likely trip existing code that was not refactored to avoid the array.

Reply:
Actually no dangerous, the only issue is with primitive arrays, that were not converted to use varargs